src/components/CCIP/Chain/Chain.astro

```diff
@@ -110,13 +109,13 @@ const searchLanes = getSearchLanes({ environment })
   .networks__grid {
     display: grid;
     grid-template-columns: 1fr;
-    gap: var(--space-8x);
+    gap: var(--space-2x);
   }
 
   .tokens__grid {
     display: grid;
-    grid-template-columns: 1fr 1fr;
-    gap: var(--space-8x);
+    grid-template-columns: 32% 32% 32%;
```

Comment:
why use the percentage instead than the FR?

Reply:
Well the grid doesn't respect the page width I tried to force it but it didn't work

Here is the current result
<img width="452" alt="Screenshot 2024-10-23 at 4 23 35 PM" src="https://github.com/user-attachments/assets/bfb33a66-23c2-472e-bf2f-5a8ac18808cd">

Here is the result with `1fr 1fr 1fr`
<img width="452" alt="Screenshot 2024-10-23 at 4 24 43 PM" src="https://github.com/user-attachments/assets/6245987c-9d0b-4f88-961c-acd7018cca47">


So the 32% is the perfect percentage to make the grid looks good in almost every mobile screen size, we are still using 1fr for the desktop

https://github.com/user-attachments/assets/2afab58a-b7de-4e85-859a-bd97df220af3

Reply:
I do not have your results. For me using `fr` works fine:
<img width="505" alt="image" src="https://github.com/user-attachments/assets/2ddcf1c6-e81d-48b4-bc74-625cec22d12a">

Reply:
Oh I tried it as well and it didn't make the same issue, probably it was a problem with the layout and I fixed it while modifying it

Reply:
I found out what this is. It is a bug with "display: contents" that is applied when using Astro islands. I reverted it back to 32%